app/views/books/_form.html.slim

```diff
@@ -0,0 +1,54 @@
+= simple_form_for book do |f|
+
+  = f.input :title, label: 'Title',
+                    placeholder: 'Title',
+                    input_html: { class: 'form-control' },
+                    wrapper_html: { class: 'form-group book-form' }
+
+  = f.input :publishing_house, label: 'Publishing house',
+                               placeholder: 'Publishing house',
+                               input_html: { class: 'form-control' },
+                               wrapper_html: { class: 'form-group book-form' }
+
+  = f.input :language, label: 'Language',
+                       placeholder: 'Language',
+                       input_html: { class: 'form-control' },
+                       wrapper_html: { class: 'form-group book-form' }
+
+  = f.input :year_of_publication, label: 'Year of publication',
+                                  placeholder: 'Year of publication',
+                                  input_html: { class: 'form-control' },
+                                  wrapper_html: { class: 'form-group book-form' }
+
+  = f.input :number_of_pages, label: 'Number of pages',
+                              placeholder: 'Number of pages',
+                              input_html: { class: 'form-control' },
+                              wrapper_html: { class: 'form-group book-form' }
+
+  = f.input :format, label: 'Format',
+                     placeholder: 'Format',
+                     input_html: { class: 'form-control' },
+                     wrapper_html: { class: 'form-group book-form' }
+
+  = f.input :binding, label: 'Binding',
+                      placeholder: 'Binding',
+                      input_html: { class: 'form-control' },
+                      wrapper_html: { class: 'form-group book-form' }
+
+  = f.input :isbn, label: 'ISBN',
+                   placeholder: 'ISBN',
+                   input_html: { class: 'form-control' },
+                   wrapper_html: { class: 'form-group book-form' }
+
+  = f.input :availability, as: :boolean,  label: 'Availability',
```

Comment:
field edit access is not offered for regular users
look at [`policies`](https://github.com/varvet/pundit#policies)
```
<% if policy(@post).update_field?(:published) %>
  <%= link_to "Edit post", edit_post_path(@post) %>
<% end %>

#post_policy.rb
def update_field?(field)
  permitted_attributes.member?(field.to_sym)
end

def permited_attributes
  ...
end
```